docs/docs.css

```diff
@@ -17,10 +17,11 @@ aside{
 }
 .container__left nav{
```

Comment:
use proper naming convention for naming css classes as you have used both naming method like using double underscore and dash

Reply:
+1 on this, chose either one and stay consistant in whole codebase

Reply:
Sure will follow one convention and replace this thing
thanks for pointing that out 👍